docs/release-notes/Percona-Everest-1.1.0-(2024-08-02).md

```diff
@@ -0,0 +1,127 @@
+
+Percona Everest is an open source cloud native database platform that helps provision and manage databases faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 1.1.0 introduces the following changes:
+
+
+## Release highlights
+
+!!! warning
+    Percona Everest 1.1.0 comes with certain [limitations](Percona-Everest-1.1.0-(2024-08-02).md#known-limitations) that you need to take into account before you update to this version.
+
+### Enhancements in the functionality for PostgreSQL backups and schedules
+
+We're thrilled to announce that we have enhanced our on-demand backups and schedules to address certain limitations with PostgreSQL. These improvements are designed to offer our users a more seamless experience when utilizing these features.
+
+Check out these improvements:
+
+- If you create more than three backup storages, you can use any existing storages across on-demand backups and schedules as long as the total number of storages in use (by existing on-demand backups and schedules) does not exceed three.
+ 
+
+    ???+ example "Example"
+        If you have created two schedules using backup storage bucket-1 and bucket-2, and an on-demand backup using backup storage bucket-3, you can only utilize one of these three backup storages to create the next on-demand backup or a schedule.
+
+
+    ![!image](../images/backup_storages.png)
+
+    ![!image](../images/pg_limitation_2.png)
+
+
+    ![!image](../images/on_demand_limitation_2.png)
+
+- You can only create up to three backup schedules for PostgreSQL.
+
+    ![!image](../images/max_three_schedules_pg.png)
+
+- You cannot change the storage location in existing schedules.
+
+- You cannot use the same storage location for multiple backup schedules.
+
+
+
+## Improvements
+
+- [EVEREST-1259](https://perconadev.atlassian.net/browse/EVEREST-1259) - We have set up a rate limiter to restrict the number of requests you can make to the API within a specific time frame. On the login page, an error will be thrown if you surpass this limit of requests within the given period.
+
+- [EVEREST-1134](https://perconadev.atlassian.net/browse/EVEREST-1134) -Starting with Percona Everest 1.1.0, you now have the convenience of upgrading the database version directly from the Namespaces page, without having to go through the hassle of using the edit DB wizard.
+
+- [EVEREST-1153](https://perconadev.atlassian.net/browse/EVEREST-1153) - We have revamped the CLI UX for install, upgrade, and uninstall commands. Prior to this, the screen was cluttered with logs that were hard to understand. But now, we have made this concise with loading animation and spinners.
+
+- [EVEREST-1088](https://perconadev.atlassian.net/browse/EVEREST-1088) -  We've removed the icons from the **Technology** column in the database list table to make it visually appealing.
```

Comment:
I wish it was more visually appealing but it is definitely not. :cry:  This was for legal reasons...

Reply:
Done